e2e/terraform/packer/linux/provision.sh

```diff
@@ -59,14 +65,46 @@ install_from_release() {
     curl -sL --fail -o /tmp/nomad.zip "$RELEASE_URL"
     sudo unzip -o /tmp/nomad.zip -d "$INSTALL_DIR"
     set_ownership
-    if [ $START == "1" ]; then sudo systemctl restart nomad; fi
 }
 
 set_ownership() {
     sudo chmod 0755 "$INSTALL_PATH"
     sudo chown root:root "$INSTALL_PATH"
 }
 
+sym() {
+    find "$1" -maxdepth 1 -type f -name "$2" 2>/dev/null \
+        | sudo xargs -I % ln -fs % "$3"
+}
+
+install_config_profile() {
+
+    if [ -d /tmp/custom ]; then
+        rm -rf /opt/config/custom
+        sudo mv /tmp/custom /opt/config/
+    fi
+
+    sudo rm -rf /etc/nomad.d/*
+    sudo rm -rf /etc/consul.d/*
+    sudo rm -rf /etc/vault.d/*
```

Comment:
I'd suggest deleting the folder and mkdir them again.  `*` has its own quarks (e.g. dot files, large dirs).

Reply:
Good catch. Will fix.